src/pauses.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Returns the position and types of pauses (vishraams) in a gurbani line
+ *
+ * @since 1.0.0
+ * @param {string} words The string from which to get the pause positions
+ * @param {string} large the symbol to use for a large pause (vishraam)
+ * @param {string} medium the symbol to use for a medium pause (jamki)
+ * @param {string} small the symbol to use for a small pause (thamki)
+ * @returns {JSON} Returns the position and type for each pause
+ * @example
+ *
+ * vishraams('ikv. sicAwrw hoeIAY; ikv. kUVY qutY pwil ]');
+ * // => '[{"p": 0, "t": "t"}, {"p": 2, "t": "v"}, {"p": 3, "t": "t"}]'
+ */
+
+
+function pauses(words = '', large = ';', medium = ',', small = '.') {
```

Comment:
guessing this won't handle larivaar?

Reply:
the word count index requires input to be padh chhed, but we should add a 'non-whitespace character count' option too; this would prevent misplaced vishraams when padh chhed differs, eg. ਪਰਮਾਦਿ ਪੁਰਖਮਨੋਪਿਮੰ ਸਤਿ ਆਦਿ ਭਾਵ ਰਤੰ vs ਪਰਮਾਦਿ ਪੁਰਖ ਮਨੋਪਿਮੰ ਸਤਿ ਆਦਿ ਭਾਵ ਰਤੰ. Either way the vishraam is placed after ਮਨੋਪਿਮੰ, the only tricky part is character counts will differ between unicode and ascii.

Reply:
@ManjotS

Reply:
can you automatically convert it to ascii to do the character count? and then use that as your index?

Reply:
Yeah that could work. Either that or reject non-ascii input. Unicode would use an updated word count based on characters. Will add this in.

Reply:
any update here @irvanjitsingh

Reply:
Will update this PR within the week

Reply:
*better late than never* Added character count support, with a note about the discrepancy between ASCII and unicode text. Resolving the discrepancy should be the client's responsibility.